src/react/workflow/__tests__/ExpirationForm.test.tsx

```diff
@@ -216,4 +217,57 @@ describe('ExpirationForm', () => {
     const formValidation = screen.getByTestId('form-expiration');
     expect(formValidation.textContent).toBe('form-valid');
   });
+
+  it('should validate ISV bucket confirmation checkbox', async () => {
```

Comment:
We should add another test to check that the checkbox is not there and the form can be submitted for non isv buckets